people_tool/static/people_tool/js/controllers/SearchController.js

```diff
@@ -98,9 +98,23 @@
             // can still display a clickable anchor link
             var lastName = (full.name_last.trim().length > 0) ?
                 full.name_last : '(none)';
-            return '<a href="' + '#/people/' + full.univ_id + '/courses/">'
+            return '<a href="' + '#/people/' + full.univ_id + '/courses/" ' +
+                'ng-click="setSelectedPersonInfo(\''+full.name_last+'\',\''+
+                full.name_first+'\','+full.univ_id+',\''+full.email_address+'\''+',\''+full.role_type_cd+'\')">'
                 + lastName + '</a>';
         };
+        $scope.setSelectedPersonInfo =
+            function(name_last, name_first, univ_id, email, role_type_cd){
```

Comment:
I wonder if there's not some way to just pass this the row's `full` object, which contains all these things in all the right keys, to reduce explicit param passing. Maybe the service code itself can selectively remember the keys it cares about. Takes some of this stuff out of the controller and moves it into the service, which would be a good practice for reusability...? I may be missing some limitations with that, though.